src/main/java/teammates/ui/pagedata/InstructorFeedbackResultsPageData.java

```diff
@@ -1044,10 +1048,7 @@ private void buildTableColumnHeaderForRecipientQuestionGiverView(List<ElementTag
         List<String> possibleReceiversWithoutResponsesForGiver = new ArrayList<>();
 
         String prevGiver = "";
-        int responseRecipientIndex = 0;
-        int responseGiverIndex = 0;
-        int userIndex = 0;
-        Map<String, Integer> userIndexesForComments = new HashMap<String, Integer>();
+        Map<String, Integer> userIndexesForComments = new HashMap<>();
```

Comment:
Same things apply here.

Reply:
This one needs to be declared here because it is used in loop and loop starts after this statement.